$NetBSD$

Fix build on DragonFly after header moved.
Do not set HAS_USL_VTS on any SunOS platform.

--- hw/xfree86/os-support/xf86_OSlib.h.orig	2011-10-28 15:02:52.128328912 +0000
+++ hw/xfree86/os-support/xf86_OSlib.h
@@ -140,7 +140,7 @@
 #  include <sys/mmap.h>		/* MMAP driver header */
 # endif
 
-# if !defined(sun) || (!defined(sparc) && !defined(__SOL8__))
+# if !defined(sun)
 #  define HAS_USL_VTS
 # endif
 # if !defined(sun)
@@ -346,7 +346,7 @@
 #    include <dev/wscons/wsdisplay_usl_io.h>
 #   endif /* WSCONS_SUPPORT */
 #   if defined(__FreeBSD__) || defined(__FreeBSD_kernel__) || defined(__DragonFly__)
-#    if defined(__FreeBSD_kernel_version) && (__FreeBSD_kernel_version >= 500013)
+#    if (defined(__FreeBSD_kernel_version) && (__FreeBSD_kernel_version >= 500013)) || (defined(__DragonFly_version) && (__DragonFly_version >= 200900))
 #     include <sys/mouse.h>
 #    else
 #     undef MOUSE_GETINFO
